munit/js/src/main/scala/java/nio/file/Files.scala

```diff
@@ -24,4 +31,59 @@ object Files {
     }
     result
   }
+
+  def createDirectories(dir: Path): Path = JSIO.inNode {
+    JSFs.mkdirSync(dir.toString(), js.Dynamic.literal(recursive = true))
+    dir
+  }
+
+  def createTempDirectory(prefix: String): Path = JSIO.inNode {
+    Paths.get(JSFs.mkdtempSync(JSPath.join(JSOS.tmpdir(), prefix)))
+  }
+
+  def exists(path: Path): Boolean = JSIO.inNode {
+    JSIO.exists(path.toString)
+  }
+
+  def deleteIfExists(path: Path): Boolean = JSIO.inNode {
+    if (JSIO.exists(path.toString)) {
+      JSFs.unlinkSync(path.toString)
+      true
+    } else {
+      false
+    }
+  }
+
+  def write(path: Path, bytes: Array[Byte], options: OpenOption*): Path =
+    JSIO.inNode {
+      val len = bytes.length
+      val data = new js.Array[Short]()
+      var curr = 0
+      while (curr < len) {
+        data(curr) = bytes(curr).toShort
+        curr += 1
+      }
+      JSFs.writeFileSync(
+        path.toString(),
+        Uint8Array.from(data),
+        js.Dynamic.literal(flag = openOptionsToFlag(options).orUndefined)
+      )
+      path
+    }
+
+  def openOptionsToFlag(options: Seq[OpenOption]): Option[String] = {
```

Comment:
For compatibility with JDK APIs
```suggestion
  private def openOptionsToFlag(options: Seq[OpenOption]): Option[String] = {
```